### ISSUE TITLE
Seg_PAHDataset

### ISSUE THREAD

**Issue by SekeunKim** [xmed-lab/CardiacNet#1]
Could you please upload Seg_PAHDataset?

infos = np.load(f'/home/jyangcu/Dataset/dataset_pa_iltrasound_nii_files_3rdcenters/save_infos_reg_v3.npy', allow_pickle=True).item()
        train_dataset_normal = Seg_PAHDataset(args, infos, set_select=['gy','rmyy','shph','szfw'], view_num=['4'], data_type=['normal'])
        train_dataset_abnorm = Seg_PAHDataset(args, infos, set_select=['gy','rmyy','shph','szfw'], view_num=['4'], data_type=['middle', 'severe'])
        train_loader_normal = DataLoader(train_dataset_normal, batch_size=args.batch_size, shuffle=True, num_workers=8)

        self.train(args, train_loader_normal, train_dataset_abnorm)

Thank you for great work.
